examples/container-amd/js/main.js

```diff
@@ -154,12 +132,6 @@ require(
 			 * Init Container
 			 */
 			F2.init({
-				UI:{
-					Mask:{
-						loadingIcon:'./img/ajax-loader.gif'
-					}
-				},
-				supportedViews: [F2.Constants.Views.HOME, F2.Constants.Views.SETTINGS, F2.Constants.Views.REMOVE],
 				secureAppPagePath: 'secure.html' // this should go on a separate domain from index.html
```

Comment:
v2 Doesn't support secure apps, so we can remove references to `secureAppPagePath`, `secure.html`, etc.